tools/protobuf/Common/IceRpc.CaseConverter.Internal/Converter.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) ZeroC, Inc.
+
+using System.Text;
+
+namespace IceRpc.CaseConverter.Internal;
+
+internal static class Converter
+{
+    internal static string ToPascalCase(this string value)
```

Comment:
Should we use instead ToTitleCase?

https://learn.microsoft.com/en-us/dotnet/api/system.globalization.textinfo.totitlecase?view=net-7.0

Reply:
I don't think so, the title case is about capitalizing separated words, which are space delimited.